analysis/structure/uconn.go

```diff
@@ -93,66 +93,66 @@ func getUniqueConnectionsScript(conf *config.Config) (string, string, []mgo.Inde
 			},
 		},
 		{
-			{"$group", bson.D{
-				{"_id", bson.D{
+			{"$group", bson.M{
```

Comment:
Yeah, this is just way easier to read and test since its closer to the mongo shell format... I didn't know about it before so now I'm trying to implement it whenever I change a query. In this case, the query actually had the bson.M thing in the first few lines anyway (my bad from a previous pull request :) ), this just made it consistent.